docker-compose.override.yml.dist

```diff
@@ -2,6 +2,7 @@ version: "3.1"
 
 services:
     db:
+        user: "${uid}:${gid}"
```

Comment:
How about this: comment these lines and in the `docker-compose.override.yml` target check if the os is linux. If yes, remove the comment and place the uid-gid pair of the current user there. That way the make file stays clean and the override file contain actual values, thus it works without make as well (with plain docker-compose commands).

I highly doubt multiple users would use the same workspace.

Reply:
I really like the idea, thank you! I added the changes accordingly. I also added a comment stating `needed for Linux users` above the `# user: "${uid}:${gid}"` lines, just in case later someone would see these commented lines and would just want to delete them. Hopefully, with this comment, they'll think twice before deleting it.